.github/workflows/ci.yml

```diff
@@ -128,3 +128,23 @@ jobs:
           cmake .. -G "MinGW Makefiles" -DBUILD_TESTING=ON -DBUILD_TESTING_INTEGRATION=ON -DC_WARNINGS_AS_ERRORS=ON -DCPP_WARNINGS_AS_ERRORS=ON
           cmake --build .
           ctest --verbose -R "allocator|value|example_basic|integration_basic"
+
+  build_and_test_linux_wasm:
+    strategy:
+     matrix:
+        platform: [ubuntu-18.04]
+    runs-on: ${{ matrix.platform }}
+    env:
+      MG_VERSION: "1.4.0"
```

Comment:
You are not using this env variable, you can remove it

Reply:
Done! Nice catch!